python/ray/tests/test_advanced_2.py

```diff
@@ -137,7 +137,9 @@ def method(self):
     assert valid_node.unique_id == ray.get(a.method.remote())
 
 
-def test_fractional_resources(shutdown_only):
+@pytest.mark.parametrize("ACCELERATOR_TYPE", ["CUDA", "XPU"])
+def test_fractional_resources(shutdown_only, ACCELERATOR_TYPE):
+    os.environ["RAY_ACCELERATOR"] = ACCELERATOR_TYPE
```

Comment:
I am concerned that these tests add more runtime and cost than the value they add. Instead of making all of these schedule tests use both CUDA and XPU, can we instead have a test which validates basic scheduling functionality across device types?

The concern is that as more accelerators are added to Ray, the runtime/cost of these tests increases linearly while 90% of the code they test is the same.